### PR TITLE
update debug_single_process to false

### PR DESCRIPTION
**Pull request by zpcore**

### CODE REVIEW

File: test/test_train_mp_imagenet.py [hunk 1]
```diff
@@ -374,5 +374,5 @@ def _mp_fn(index, flags):
 
 if __name__ == '__main__':
   # if running with torchrun, nprocs argument will be omitted.
-  debug_single_process = True if FLAGS.num_cores == 1 else False
-  torch_xla.launch(_mp_fn, args=(FLAGS,), debug_single_process=True)
+  do_debug = True if FLAGS.num_cores == 1 else False
```
Comment (will-cromar) [inline test/test_train_mp_imagenet.py:377]: Can you keep the name `debug_single_process`?

**Review by will-cromar — APPROVED** [review]
Thanks!

**Comment by miladm** [comment]
qq: is there a plan to migrate all such API calls? (e.g. https://github.com/pytorch/xla/blob/master/test/test_train_mp_mnist.py#L220) @zpcore

**Comment by zpcore** [comment]
> qq: is there a plan to migrate all such API calls? (e.g. https://github.com/pytorch/xla/blob/master/test/test_train_mp_mnist.py#L220) @zpcore

Yes, I will make a follow up PR to apply torch_xla.launch() to all tests and update the doc. @will-cromar and @JackCaoG  to see if you have any concerns.

**Pull request merged**